.config/.env.example

```diff
@@ -1,22 +1,25 @@
 # Database Config
 POSTGRES_HOST=
 POSTGRES_PORT=
-POSTGRES_DB=db_dev
-POSTGRES_USER=admin
+POSTGRES_DB=
+POSTGRES_USER=
 POSTGRES_PASSWORD=
 
-# Logging database config
```

Comment:
I'm going to be opening a PR soon with some README fixes, so I'll add these comments back in then :)